src/components/OptionsSelector.js

```diff
@@ -220,8 +220,8 @@ class OptionsSelector extends Component {
             allOptions: newOptions,
             focusedIndex: newFocusedIndex,
         }, () => {
-            // If we just selected a new option on a multiple-selection page, scroll to the top
-            if (this.props.selectedOptions.length > prevProps.selectedOptions.length) {
+            // If we just selected or deselected an option on a multiple-selection page, scroll to the top
```

Comment:
idk, less words is better?

```suggestion
            // If we just toggled an option on a multiple-selection page, scroll to the top
```

Reply:
`// If we just toggled an option on a multi-selection page, scroll to top`
@rushatgabhane is this better

Reply:
cool cool, please ping me whenever you've added the videos.
Thanks so much! 🙇‍♂️